tests/refl.py

```diff
@@ -256,9 +266,11 @@ def test_GIVEN_move_was_issued_while_different_move_already_in_progress_WHEN_mov
         self.ca_galil.assert_that_pv_is("MTR0102.DMOV", 1, timeout=30)
         self.ca_galil.assert_that_pv_is("MTR0102.VELO", expected)
 
-    def test_GIVEN_move_in_progress_WHEN_modifying_motor_velocity_THEN_motor_retains_new_value_after_move_completed(self):
+    def test_GIVEN_move_in_progress_WHEN_modifying_motor_velocity_THEN_velocity_reverted_to_value_before_modified_velocity(self):
```

Comment:
Currently, this test is meaningless. At line 278:

```
        self.ca_galil.set_pv_value("MTR0102.VELO", expected)
```

you need to assign a different value than the initial one.